docs/forgeops_doctor.md

```diff
@@ -34,6 +35,6 @@ Diagnose common cluster and platform deployments
 ### SEE ALSO
 
 * [forgeops](forgeops.md)	 - forgeops is a tool for managing ForgeRock platform deployments
-* [forgeops doctor operator](forgeops_doctor_operator.md)	 - Verify that operators are installed and ready
+* [forgeops doctor operator](forgeops_doctor_operator.md)	 - Check Operators Installed and Running
```

Comment:
If this ends up in help text, revert it to what it was (but change "ready" to "running")